dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layerparam/initializer/PoolingLayerParameterInitializer.java

```diff
@@ -74,6 +74,17 @@ public int getIndex() {
   }
 
   /**
+   * This function checks if stride is set proper for the input shape.
+   */
+  private void checkShape() {
```

Comment:
I am wondering if we cannot support this case. Caffe seems to work for this case. Can you check it?